sippy-ng/src/component_readiness/TriagedRegressionTestList.js

```diff
@@ -0,0 +1,105 @@
+import { DataGrid, GridToolbar } from '@mui/x-data-grid'
+import { formColumnName } from './CompReadyUtils'
+import { relativeTime } from '../helpers'
+import { Tooltip, Typography } from '@mui/material'
+import PropTypes from 'prop-types'
+import React, { Fragment } from 'react'
+
+export default function TriagedRegressionTestList(props) {
+  const [sortModel, setSortModel] = React.useState([
+    { field: 'component', sort: 'asc' },
+  ])
+
+  const [triagedRegressions, setTriagedRegressions] = React.useState([])
+  const [showView, setShowView] = React.useState(false)
+
+  const handleTriagedRegressionGroupSelectionChanged = (data) => {
+    let displayView = false
+    if (data) {
+      displayView = true
+    }
+    setTriagedRegressions(data)
+    setShowView(displayView)
+  }
+  props.eventEmitter.on(
+    'triagedEntrySelectionChanged',
+    handleTriagedRegressionGroupSelectionChanged
+  )
+
+  const columns = [
+    {
+      field: 'test_name',
+      headerName: 'Test Name',
+      flex: 40,
+      valueGetter: (params) => {
+        return params.row.test_name
+      },
+      renderCell: (param) => <div className="test-name">{param.value}</div>,
+    },
+    {
+      field: 'release',
+      headerName: 'Release',
+      flex: 20,
+      valueGetter: (params) => {
+        return params.row.release
+      },
+      renderCell: (param) => <div className="test-name">{param.value}</div>,
+    },
+    {
+      field: 'variants',
+      headerName: 'Variants',
+      flex: 30,
+      valueGetter: (params) => {
+        return formColumnName({ variants: params.row.variants })
+      },
+      renderCell: (param) => <div className="test-name">{param.value}</div>,
+    },
+    {
+      field: 'opened',
+      headerName: 'Regressed Since',
+      flex: 12,
+      valueGetter: (params) => {
+        if (!params.row.opened) {
+          // For a regression we haven't yet detected:
+          return ''
+        }
+        const regressedSinceDate = new Date(params.row.opened)
+        return relativeTime(regressedSinceDate, new Date())
+      },
+      renderCell: (param) => (
+        <Tooltip title="WARNING: This is the first time we detected this test regressed in the default query. This value is not relevant if you've altered query parameters from the default.">
+          <div className="regressed-since">{param.value}</div>
+        </Tooltip>
+      ),
+    },
```

Comment:
Can/Should we add a column to the test details page like we do in the Untriaged Regressions, etc.?

Reply:
I think that is a nice idea, but can it be included in a follow up? This PR has had many eyes on it at this point, and I have some more WIP that is depending on it. It will be easier to do that after.